cg/store/models.py

```diff
@@ -992,7 +992,7 @@ class RunDevice(Base):
     )
 
     @property
-    def _samples(self) -> list[Sample]:
+    def samples(self) -> list[Sample]:
```

Comment:
Can you change line 1001 while you're at it? It was the bug we discovered ages ago. I'm pretty sure it should be `run.sample_metrics`

Reply:
done